examples/VLCB_1in1out/VLCB_1in1out.ino

```diff
@@ -205,7 +205,7 @@ void processModuleSwitchChange()
   if (moduleSwitch.stateChanged())
   {
     bool state = moduleSwitch.isPressed();
-    byte eventNumber = 1;
+    byte eventNumber = 0;  // First switch has an event index of 0
```

Comment:
Are you sure about this? 
It doesn't really matter as this example sketch can do what it wants. 
Just curious why you decided to change?
It looks a bit strange to have an EN == 0.